class/distributionlistsocpeople.class.php

```diff
@@ -414,6 +414,28 @@ public function update(User $user, $notrigger = false)
 		return $this->updateCommon($user, $notrigger);
 	}
 
+	/**
+	 * Supprimer le contact donné en paramètre de toutes les listes de diffusion
+	 *
+	 * @param User $user       User that deletes
+	 * @param User $fk_socpeople   Contact  condition
+	 * @param bool $notrigger  false=launch triggers after, true=disable triggers
+	 * @return int             <0 if KO, >0 if OK
+	 */
+	public function deleteByContact(User $user, $fk_socpeople, $notrigger = false)
+	{
+
+		$sql = 'DELETE FROM '.MAIN_DB_PREFIX.$this->table_element.' WHERE fk_socpeople = '.intval($fk_socpeople);
+		$resql = $this->db->query($sql);
+
+		if($resql){
+			return 1;
+		} else {
+			return 1;
```

Comment:
il doit manquer un "-" non ?
```suggestion
			return -1;
```